src/script.js

```diff
@@ -20,11 +20,12 @@ const attachEventListeners = (inputElement, buttonElement, amountFoundElement) =
   });
 };
 
-document.addEventListener("DOMContentLoaded", function () {
-  const references = initilizeElementsRefrences();
-  attachEventListeners(...references);
+Object.keys(global).includes('document') &&
```

Comment:
Could you elaborate on this change?

Reply:
As this function is in global scope it tries to add event listener into document on script import.
Inside tests global scope is not initialised yet so I have to check if global scope includes document scope.
In real case scenario it will run without problems, its just a workaround for tests global document initialisation.

Reply:
Isn't changing the code implementation to fit the test setup kinda an awful idea?